python/python/tests/test_config.py

```diff
@@ -52,3 +57,195 @@ def test_periodic_checks_interval_to_protobuf():
     config.periodic_checks = PeriodicChecksManualInterval(30)
     request = config._create_a_protobuf_conn_request(cluster_mode=True)
     assert request.periodic_checks_manual_interval.duration_in_sec == 30
+
+
+@pytest.mark.asyncio
+class TestAZAffinity:
+
+    def test_convert_config_with_azaffinity_to_protobuf(self):
```

Comment:
With the exception of this test, I believe you should move the rest of the tests either to a new file called test_read_from_strategy.py or to another suitable location.